lnd.go

```diff
@@ -713,6 +713,8 @@ func waitForWalletPassword(grpcEndpoints, restEndpoints []net.Addr,
 	// Set up a new PasswordService, which will listen for passwords
 	// provided over RPC.
 	grpcServer := grpc.NewServer(serverOpts...)
+	// This server is not used outside this function
```

Comment:
Nit: I think the `defer` already implies this comment, so could be removed.

Reply:
Yeah, comment can be removed.